README.md

```diff
@@ -24,18 +24,29 @@ Ideas for future features include:
 
 ## How to run locally
 
+Install Maven if you dont have it pre-installed . [See here](https://www.mkyong.com/maven/how-to-install-maven-in-windows/)
+
 You need the Maven settings.xml from OpenDaylight to have Maven grab stuff 
 from https://nexus.opendaylight.org instead of from https://repo.maven.apache.org/maven2.
 This is [described in the OpenDaylight documentation](http://docs.opendaylight.org/en/latest/developer-guide/developing-apps-on-the-opendaylight-controller.html),
 but basically just:
 
     cp -n ~/.m2/settings.xml{,.orig} ; wget -q -O - https://raw.githubusercontent.com/opendaylight/odlparent/master/settings.xml > ~/.m2/settings.xml
 
+As an alternative you can copy the xml code from [settings.xml](https://raw.githubusercontent.com/opendaylight/odlparent/master/settings.xml) and simply paste it into `C:\Users\user_name\.m2\settings.xml`
+
 then you can:
 
     ./mvnw package
     java -jar spring/target/org.opendaylight.bot.spring.jar
 
+or 
```

Comment:
may be say something like "or, during ongoing WIP local development BEFORE pushing to GitHub for Pull Request (but do run CS locally before raising a PR; pull request with build failures will be ignored):